v3/plugins/atlassian/atlascode/2.6.1/meta.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+publisher: atlassian
+name: atlascode
+version: 2.6.1
+type: VS Code extension
+displayName: Jira and Bitbucket
+title: Jira and Bitbucket Plugin
+description: Bringing the power of Jira and Bitbucket to VS Code - With Atlassian for VS Code you can create and view issues, start work on issues, create pull requests, do code reviews, start builds, get build statuses and more!
+icon: https://bitbucket.org/atlassianlabs/atlascode/raw/1b318e660ca1b8091a9b433de935b099ba71d097/images/atlascode-icon.png
+category: Other
+repository: https://bitbucket.org/atlassianlabs/atlascode
+firstPublicationDate: "2020-05-26"
+spec:
+  extensions:
+  - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/Atlassian/vsextensions/atlascode/2.6.1/vspackage
```

Comment:
Asked guys about that
https://bitbucket.org/atlassianlabs/atlascode/issues/303/make-atlascode-compatible-for-theia

Reply:
I think it's better to build and upload the binary on download.jboss.org rather than refer to https://bitbucket.org/atlassianlabs/atlascode. We cannot guarantee that guys will always update the extension with each new lerease.

Reply:
Agreed, let's build + publish it on download.jboss.org.

Reply:
where are we on having it on jboss.org ?

Reply:
With all the other extensions we've built: https://download.jboss.org/jbosstools/vscode/3rdparty/

We can't link to the VS Code Marketplace. @vitaliy-guliy is there anything blocking this plugin now? Can we build it and close this PR?

Reply:
Working on it right now. Adding build to the Jenkins CI.